### PR TITLE
Update README.md for using new binary image.

### PR DESCRIPTION
**Pull request by enriquebelarte**
Leave e2e image to e2e tests and use new ocm-tf-bin image for testing binary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,11 +77,9 @@ If you want to build a local OCM provider to develop improvements for the Red Ha
...
-    
-    <HOME>/.terraform.d/plugins/terraform.local/local/ocm/<VERSION>/<TARGET_ARCH>
-    
-to your local using the following example: 
-
-    podman run --pull=always --rm registry.ci.openshift.org/ci/ocm-tf-e2e:latest cat /root/.terraform.d/plugins/terraform.local/local/ocm/1.0.1/linux_amd64/terraform-provider-ocm > ~/terraform-provider-ocm && chmod +x ~/terraform-provider-ocm
-    can also use specific commit images by substituting `latest` for the desired commit SHA.
+If you want to locally test the provider binary without building from sources, you can pull the `latest` binary container image and copy the binary file to your local:
+```
+    podman run --pull=always --rm registry.ci.openshift.org/ci/ocm-tf-bin:latest cat /root/terraform-provider-ocm > ~/terraform-provider-ocm && chmod +x ~/terraform-provider-ocm
```
Comment (bardielle) [inline README.md:82]: maybe we can add it as a make command?

**Comment by sagidayan** [comment]
CI failed due to quota on AWS. We should try again later.

File: Makefile [hunk 2]
```diff
@@ -121,3 +121,7 @@ apply_folder: install
 .PHONY: destroy_folder
 destroy_folder: install
 	bash ./ci/destroy_folder.sh
+
+.PHONY: binary
+binary:
+	podman run --pull=always --rm registry.ci.openshift.org/ci/rhcs-tf-bin:latest cat /root/terraform-provider-ocm > ~/terraform-provider-ocm && chmod +x ~/terraform-provider-ocm
```
Comment (nirarg) [inline Makefile:127]:
```suggestion
.PHONY: binary
binary:
	podman run --pull=always --rm registry.ci.openshift.org/ci/rhcs-tf-bin:latest cat /root/terraform-provider-rhcs > ~/terraform-provider-rhcs && chmod +x ~/terraform-provider-rhcs
```

File: README.md [hunk 3]
```diff
@@ -94,12 +94,9 @@ If you want to build a local Red Hat Cloud Services provider to develop improvem
...
-    <HOME>/.terraform.d/plugins/terraform.local/local/rhcs/<VERSION>/<TARGET_ARCH>
-    
-Paste it to your local using the following example: 
-
-    podman run --pull=always --rm registry.ci.openshift.org/ci/rhcs-tf-e2e:latest cat /root/.terraform.d/plugins/terraform.local/local/rhcs/1.0.5/linux_amd64/terraform-provider-rhcs > ~/terraform-provider-rhcs && chmod +x ~/terraform-provider-rhcs
-    
+If you want to locally test the provider binary without building from sources, you can pull the `latest` binary container image and copy the binary file to your local by running `make binary` or the whole background command if you need to make custom changes:
+```
+    podman run --pull=always --rm registry.ci.openshift.org/ci/rhcs-tf-bin:latest cat /root/terraform-provider-ocm > ~/terraform-provider-ocm && chmod +x ~/terraform-provider-ocm
+```
```
Comment (nirarg) [inline README.md:100]:
```suggestion
    podman run --pull=always --rm registry.ci.openshift.org/ci/rhcs-tf-bin:latest cat /root/terraform-provider-rhcs > ~/terraform-provider-rhcs && chmod +x ~/terraform-provider-rhcs
```
```

File: Makefile (hunk 2 above)
Comment (enriquebelarte) [reply Makefile]: Thank you!

**Review by sagidayan — APPROVED** [review]

**Pull request merged**